core/src/trezor/workflow.py

```diff
@@ -187,7 +205,11 @@ def touch(self) -> None:
         """
```

Comment:
This docstring probably needs an update. USB messages no longer count as activity.

Reply:
Also please document `restore_deadline`.

Reply:
Still trying to understand this. In practice there is only one task in `self.tasks`, the `lock_device_if_unlocked()`, right? But the function is written like there could be more tasks, each one with it's own timeout. Now we are somehow mixing up all the deadlines together with `storage.cache.autolock_deadline` even though they can be different?